src/main/resources/templates/index.html

```diff
@@ -20,21 +20,6 @@
 
 </div>
 
-<form action="#" th:action="@{/process_register}" th:object="${user}" enctype="multipart/form-data"  method="POST"><br />
-    <div class="input-group input-group-sm mb-3">
-        <div class="input-group-prepend">
-            <span class="input-group-text" id="Username">Username</span>
-        </div>
-        <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="Username" th:field="*{userName}"/>
-    </div>
-    <div class="input-group input-group-sm mb-3">
-        <div class="input-group-prepend">
-            <span class="input-group-text" id="Password">Password</span>
-        </div>
-        <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="Password" th:field="*{password}"/>
-    </div>
-    <button name="submit" type="submit" value="0">Submit</button>
-</form>
```

Comment:
I removed the from from this page to its own page. I think the form as it is was out of place and it was taking up space.